src/stylesheets/components/_page-navigation.scss

```diff
@@ -6,7 +6,7 @@
 }
 
 .app-page-navigation__item {
-  @include govuk-font(19);
+  @include govuk-typography-common;
```

Comment:
This one's very subtle but the list dashes become shorter because the em dash character is falling back to arial:

With `govuk-typography-common`:
<img width="274" alt="Screenshot of typography page navigation with govuk-typography-common applied to the dashes in the list items" src="https://github.com/alphagov/govuk-design-system/assets/64783893/5272d9b5-2c3f-4f19-9724-1ddb02a5e114">

Without:
<img width="286" alt="Screenshot of typography page navigation without govuk-typography-common applied to the dashes in the list items" src="https://github.com/alphagov/govuk-design-system/assets/64783893/509029be-b958-4295-b92f-8252ab69d836">

It's unclear to me how much of an issue this is. I suspect if we got rid then the consequences would be minimal to none.

Reply:
Ah nice spot, keep it as you've suggested then. Thanks for checking